binder/views.py

```diff
@@ -1422,7 +1449,10 @@ def store_m2m_field(obj, field, value, request):
 
 		# Skip re-fetch and serialization via get_objs if we're in
 		# multi-put (data is discarded!).
-		if getattr(request, '_is_multi_put', False):
+		if (
+			getattr(request, '_is_multi_put', False) or
+			getattr(request, '_is_file_upload', False)
```

Comment:
Does this mean we do not get the response in json anymore of add files to our request. Is there any advantage in not doing it?

Reply:
This check is about file uploads through dispatch file field, not the new file uploads. Added a comment to clarify.